assets/locales/cs.json

```diff
@@ -29,12 +29,12 @@
     "east": "Východní",
     "extent": "Rozsah",
     "fill": "Výplň",
-    "fillInDescriptive": "Vyplňte popisný text o mapě",
-    "fillInName": "Vyplňte jméno",
+    "fillInDescriptive": "Vyplňte stručný popis mapy",
+    "fillInName": "Vyplňte název",
     "filter": "Filtr:",
     "folder": "Složka",
-    "from": "Z",
-    "fullText": "Full text",
+    "from": "Od",
+    "fullText": "Plný text",
```

Comment:
Depends on the context. I suppose it does mean fulltext search. It might be better to use "Hledat v textu" or left "Full text" there.

Reply:
Right. Don't know if the Czech equivalent is in common use. Probably not.. I like "Hledat v textu" in this context.

Reply:
This string is used in the advanced-micka-dialog where "fulltext" seemed more appropriate in the end...